src/printer.js

```diff
@@ -2256,6 +2256,12 @@ function isEmptyJSXElement(node) {
   }
 }
 
+function isJSXWhitespace(node) {
```

Comment:
kind of iffy on this, would be preferable to have one `jsxWhitespace` object, set once, that could be compared `===`. Couldn't think of a quick way to do so, figured this approach was fine.